app/helpers/risks_helper.rb

```diff
@@ -1,14 +1,6 @@
 # frozen_string_literal: true
 
 module RisksHelper
-  def heightened_observation_yes_no(level)
-    if Forms::Risk::RiskToSelf::OBSERVATION_LEVELS_REQUIRING_DETAILS.include?(level)
-      'yes'
-    else
-      'no'
-    end
-  end
-
   def full_observation_level_details(level, details)
     return '' if level == Forms::Risk::RiskToSelf::OBSERVATION_LEVELS.first
```

Comment:
This line:

```
return '' if level == Forms::Risk::RiskToSelf::OBSERVATION_LEVELS.first
```

in method full_observation_level_details should be now removed as a result of your work